cosmos/dbt/graph.py

```diff
@@ -166,7 +166,7 @@ def run_command(command: list[str], tmp_dir: Path, env_vars: dict[str, str]) ->
     process = Popen(
         command,
         stdout=PIPE,
-        stderr=PIPE,
+        stderr=STDOUT,
```

Comment:
I can see a few downsides to this implementation:
1. Loss of stream separation: it is not possible to distinguish between regular output and error messages
2. Unnecessary memory overhead: combining stdout and stderr, if in large volume, can lead to memory overhead
3. Error handling: if we want to handle stderr and stdout logs differently, we cannot

Ordering chronologically the dbt logs between stdout and stderr does not justify these downsides.

I was okay with the original implementation - and once the tests are updated, the CI checks pass, I can approve and merge this change.

Reply:
Got it.